lib/update/internal/rollingupdate/dispatcher.go

```diff
@@ -51,6 +51,10 @@ func (r *dispatcher) Dispatch(config Config, params fsm.ExecutorParams, remote f
 		return libphase.NewUncordon(params, config.Client, logger)
 	case libphase.Endpoints:
 		return libphase.NewEndpoints(params, config.Client, logger)
+	case libphase.Custom:
+		// Custom phase is not defined by default. It can be optionally provided
+		// by custom dispatcher implementations
+		return nil, nil
```

Comment:
If it's defined by other implementations, does it need to be defined here as well? Just thinking, could there be a case where the caller will panic since this here returns (nil, nil)?